example/index.json

```diff
@@ -0,0 +1,11 @@
+{
+  "key": "payrue",
```

Comment:
You shouldn't change the folder "example", instead you should make your own folder "payrue"